dhctl/pkg/kubernetes/actions/manifests/manifests_test.go

```diff
@@ -81,36 +79,3 @@ func Test_struct_vs_unmarshal(t *testing.T) {
 
 	fmt.Printf("%d lines are differ\n", diff)
 }
-
-func Test_DeployTime(t *testing.T) {
```

Comment:
Why do you delete this test?

We added it, because in previous version we got new time for every `bootstrap` running and it led to restarts Deckhouse pod.

I think we should stay test and fix logic (see [this comment](https://github.com/deckhouse/deckhouse/pull/716/files#r796348766))

Reply:
Good catch, Nikolay. It was an accedent. I reverted these changes.